pkg/clients/handlers.go

```diff
@@ -220,7 +240,7 @@ func NewAPI(service ServiceInterface, logger logging.LoggerInterface) *API {
 	a := new(API)
 
 	a.service = service
-
+	a.validator = validator.New(validator.WithRequiredStructEnabled())
```

Comment:
any benefit in having this passed in as a dependency and share it across the whole app?